.github/ISSUE_TEMPLATE/crash.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Crash
+about: Did OpenTTD crash?
+title: "Crash Report"
+---
+<!-- Please zip the crash.log, crash.dmp and crash.sav and attach it to this crash report. -->
+
+## Version of OpenTTD
+<!-- Indicate what version of OpenTTD you are using, including your OS. -->
+
+## Steps to reproduce
+<!-- Please spend a few words if you can reproduce this problem. -->
```

Comment:
It should be obvious and common sense and totally unnecessary, but the template does not indicate a place to describe the issue/bug.

Reply:
What do you mean? Isn't "crash" enough? "Expected behavior: not crash  Actual behavior: crash" does not sound useful to me :p

Reply:
Yeah, exactly that. I don't want more details besides: it crashes, honestly. I just want to know how they managed to do that :D If it turns out it is a horrible idea to not ask to describe it, we can always add it back in :)

Reply:
Taking a proper DAU perspective: Even if the template says to upload crash.*, doesn't mean people will manage that. And the DAU is only supposed to spend a few words if reproducible, ending up with "crash 1.11.0".

This argument assume maximum stupidity, of course 😄